docs/source/using/interacting-with-openshift.adoc

```diff
@@ -46,7 +46,7 @@ for example, logging into another OpenShift instance, you can run:
 $ oc config use-context minishift
 ----
 
-For an introduction to `oc` usage, refer to the
+For an introduction to `oc` usage, see the
```

Comment:
Well, is the location of the `oc` command still relevant now we have `minishift oc-env`?